docs-md/tutorials/materialized.md

```diff
@@ -0,0 +1,373 @@
+
+```
+docker exec -it mysql /bin/bash
+```
+
+And then logging into MySQL as root:
+
+```
+mysql -u root -p
+```
+
+The root password, as specified in the Docker Compose file, is `mysql-pw`.
+
+For simplicity, this tutorial grants all privileges to `example-user` connecting from any host. In the real world, you'd want to manage your permissions much more tightly.
+
+Grant the privileges for replication by executing the following statement at the MySQL prompt:
+
+```sql
+GRANT ALL PRIVILEGES ON *.* TO 'example-user' WITH GRANT OPTION;
+ALTER USER 'example-user'@'%' IDENTIFIED WITH mysql_native_password BY 'example-pw';
+FLUSH PRIVILEGES;
+```
+
+### Create the calls table in MySQL
+
+Seed your blank database with some initial state. In the same MySQL CLI, switch into the `call-center` database:
+
+```sql
+USE call-center;
+```
+
+Create a table that represents phone calls that were made. Keep this table simple: the columns represent the name of the person calling, the reason that they called, and the duration in seconds of the call.
+
+```sql
+CREATE TABLE calls (name TEXT, reason TEXT, duration_seconds INT);
+```
+
+And now add some initial data. You'll add more later, but this will suffice for now:
+
+```sql
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("michael", "purchase", 540);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("michael", "help", 224);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("colin", "help", 802);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "purchase", 10204);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "help", 600);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("colin", "refund", 105);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("michael", "help", 2030);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("colin", "purchase", 800);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "help", 2514);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "refund", 325);
+```
+
+### Start the Debezium connector
+
+With MySQL ready to go, connect to ksqlDB's server using its interactive CLI. Run the following command from your host:
+
+```
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+Before you issue more commands, tell ksqlDB to start all queries from earliest point in each topic:
+
+```sql
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Now you can connect ask Debezium to stream MySQL's changelog into {{ site.ak }}. Invoke the following command in ksqlDB, which creates a Debezium source connector and writes all of its changes to {{ site.ak }} topics:
```

Comment:
`you can connect ask Debezium`? 
Should it be `you can ask Debezium`?

Reply:
Thanks!